src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsoffendercategorisationapi/model/response/catform/CatForm.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.hmppsoffendercategorisationapi.model.response.catform
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class CatForm(
+  val id: String? = null,
+
+  @JsonProperty("form_response")
+  val formResponse: Map<String, Any>? = null,
+
+  @JsonProperty("booking_id")
+  val bookingId: String? = null,
+
+  val status: String? = null,
+
+  @JsonProperty("referred_date")
+  val referredDate: String? = null,
+
+  @JsonProperty("sequence_no")
+  val sequenceNo: String? = null,
+
+  @JsonProperty("risk_profile")
+  val riskProfile: RiskProfile? = null,
+
+  @JsonProperty("prison_id")
+  val prisonId: String? = null,
+
+  @JsonProperty("offender_no")
+  val offenderNo: String? = null,
+
+  @JsonProperty("start_date")
+  val startDate: String? = null,
```

Comment:
You can declare dates as LocalDate so long as you add a @JsonFormat(pattern = "yyyy-MM-dd")

Reply:
For the dates I have just used string as for the purpose of this SAR work is just output the data from the db as is,so no need for date manipulation or formatting etc